pom.xml

```diff
@@ -69,6 +98,18 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
Why do you need `maven-install-plugin`?

Reply:
It was made to avoid using system scope for dependency. As I understand now, it doesn't work because we never make "Install" stage